test/functional/answers_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
```

Comment:
Actually I tried it but was getting some errors. So thought of testing with jasmine. I will give you the updates.